main.go

```diff
@@ -21,8 +24,12 @@ var (
 
 	configFile  string
 	versionFlag bool
+
+	autoSetup bool
 )
 
+const privateIP = "10.10.10.10"
```

Comment:
Hardcoded IP is not recommended. Sometimes it may cause conflicts.

Reply:
[-private-ip option](https://github.com/xjasonlyu/tun2socks/pull/284/commits/b672a3e14789b0fa58b54ec6f7d205f6889b26a6)